docs/BATS.md

```diff
@@ -0,0 +1,44 @@
+# BATS testing framework
+
+This is relevant starting from [v0.11.52](https://github.com/redkubes/otomi-core/releases/tag/v0.11.52).
+
+## Where can I find BATS?
+
+The otomi/tools:1.4.10 includes the [bats framework](https://github.com/bats-core/bats-core), including libraries [bats-assert](https://github.com/ztombol/bats-assert), [bats-file](https://github.com/ztombol/bats-support) and [bats-support](https://github.com/ztombol/bats-support). These links include relevant documentation, such as syntax, which won't be discussed on this page.
+
+Example of calling the binary:
+
+`docker run --rm otomi/tools:1.4.10 bats bin/tests`
+
+This example command assumes that tests are found in the container directory `otomi-core/bin/tests`. You can call bats with a directory as parameter and it will execute any file with the .bats extension. Note that these files can be found in the container directory \$WORKDIR/bin/tests.
+
+## How should I add tests?
+
+By convention, append new test files or create new test files in the \$root/bin/tests folder. Follow [this Shell Style Guide](https://github.com/google/styleguide/blob/gh-pages/shellguide.md) and apply the following rules:
+
+- use 'true' and 'false' strings for boolean flags, and always compare them as strings `[[ "$var" == "false" ]]`
```

Comment:
I think that there should be a separate document about coding guide.

Reply:
I agree, but it wasn't added so far. So do you suggest to remove the reference here altogether?

Reply:
yes